google/cloud/bigtable/tests/snapshot_async_integration_test.cc

```diff
@@ -153,7 +153,7 @@ TEST_F(SnapshotAsyncIntegrationTest, CreateListGetDeleteSnapshot) {
   EXPECT_FALSE(IsSnapshotPresent(*snapshots_after_delete, snapshot.name()));
 
   // delete table
-  EXPECT_STATUS_OK(DeleteTable(table_id.get()));
+  EXPECT_STATUS_OK(table_admin_->DeleteTable(table_id.get()));
```

Comment:
I think this test could use `GetTable()`.

Reply:
Done.